actions/mongodb_explain_action_test.go

```diff
@@ -42,34 +46,60 @@ func TestMongoDBExplain(t *testing.T) {
 	err := prepareData(ctx, client, database, collection)
 	require.NoError(t, err)
 
-	params := &agentpb.StartActionRequest_MongoDBExplainParams{
-		Dsn:   tests.GetTestMongoDBDSN(t),
-		Query: `{"ns":"test.coll","op":"query","query":{"k":{"$lte":{"$numberInt":"1"}}}}`,
-	}
+	t.Run("Valid MongoDB query", func(t *testing.T) {
+		params := &agentpb.StartActionRequest_MongoDBExplainParams{
+			Dsn:   tests.GetTestMongoDBDSN(t),
+			Query: `{"ns":"test.coll","op":"query","query":{"k":{"$lte":{"$numberInt":"1"}}}}`,
+		}
 
-	ex := NewMongoDBExplainAction(id, params)
-	res, err := ex.Run(ctx)
-	assert.Nil(t, err)
+		ex := NewMongoDBExplainAction(id, params)
+		res, err := ex.Run(ctx)
+		assert.Nil(t, err)
```

Comment:
![image](https://user-images.githubusercontent.com/11512/86606784-217a4800-bfb1-11ea-8b64-9f5925ca838a.png)

Is it `gofmt`'ed?

Reply:
Yes, I ran make format

Reply:
I wonder _how_ GitHub broke diff rendering just in that place